core/src/test/java/com/netflix/iceberg/hadoop/TestHadoopCommits.java

```diff
@@ -242,4 +246,50 @@ public void testMergeAppend() throws Exception {
     Assert.assertEquals("Current snapshot should contain 1 merged manifest",
         1, metadata.currentSnapshot().manifests().size());
   }
+
+  @Test
+  public void testSchemaUpdateComplexType() throws Exception {
+    Assert.assertTrue("Should create v1 metadata",
+            version(1).exists() && version(1).isFile());
```

Comment:
Minor: indentation seems off compared to other code in file.

Reply:
+1. Fixed indentation to 2 spaces.